### ISSUE TITLE
Should we add unknown entry for when there is missing metadata?

### ISSUE THREAD

**Issue by GeorgeKerscher** [w3c/publ-a11y#252]
In the Principles we have a note:
When the content creator does not provide any accessibility metadata for a publication, the three pieces of key information that should always be present can still be shown (with an indication that the information is missing): Supports nonvisual reading, Visual adjustments and Conformance.

First, I think we should put the order of these statements in the same order as they occur in the rest of the document, i.e., Supports Visual adjustments is first and conformance is third.

Now, should we always have a suggested statement When it is unknown? We should be careful in this language because the hazard section has a metadata value of unknown. 

Perhaps we can keep it simple in most places with unknown and in the hazard section we write:

When there is no audio in the book or when the audio hazard is none:
There are no audio hazards
or
There are audio hazards.
or
Audio hazards are unknown.
or
Audio Hazard information is missing.

OTE

**Comment by gautierchomel** [comment]
The hazard information is particular because it applies only to ebooks with multimedia content which are few. Therefore, the choice of displaying it or not should follow similar behavior as the [4.2 Extended filtering set section](https://w3c.github.io/publ-a11y/UX-Guide-Metadata/draft/principles/?updated#extended-filtering-set). 

Maybe we could reuse 4.2 information to extend the notes introducing [3. Key accessibility information](https://w3c.github.io/publ-a11y/UX-Guide-Metadata/draft/principles/?updated#order-of-key-information). If we do so, then we may want to add information about what to display in case of missing.

**Comment by avneeshsingh** [comment]
I think we should not do over thinking on hazards at this point of time.
Ultimately people will ask how to determine that certain sound or visual effect should be seen as a hazard. And the only specification we have in WCAG is for flashing hazards.

**Comment by GeorgeKerscher** [comment]
I believe we have address everything brought up in the comments.

**Comment by avneeshsingh** [comment]
OK @GeorgeKerscher , closing it for now.
